app/views/school/details/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<%- content_for :before_content do %>
+  <%= govuk_link_to('Back', school_home_path, class: 'govuk-back-link') %>
+<% end %>
+<% content_for :title, t('page_titles.school_details') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <span class="govuk-caption-xl"><%= @school.name %></span>
+      <%= t('page_titles.school_details') %>
+    </h1>
+
+    <%= render SummaryListComponent.new(rows: @school_summary) %>
+
+    <h2 class="govuk-heading-m">
+      About Google Chromebooks
+    </h2>
+
+    <p class="govuk-body">It’s possible to order a range of different devices (laptops, tablets, Chromebooks). If you are going to order Google Chromebooks, we’ll need 2 pieces of information to configure them:</p>
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
+      <li>the domain your school uses for its Google services</li>
+      <li>the school’s recovery email address</li>
+    </ul>
+    <p class="govuk-body">We need this information so we can secure the devices.</p>
+
+    <p class="govuk-body">
+      If you need help finding these details, email <a href="mailto:COVID.TECHNOLOGY@education.gov.uk">COVID.TECHNOLOGY@education.gov.uk</a>
```

Comment:
can we use the `ghwt_contact_mailto` helper here?

Reply:
done